de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties

```diff
@@ -582,6 +582,14 @@ vis_tools_note_three = Zurzeit speichern wir diese Modelle im Hauptspeicherberei
 vis_tools_note_four = Die Modelle werden bei jedem Neustart des Servers aktualisiert. Da dies auf der Produktionsinstanz  allgemein unpraktisch ist, können AdministratorInnen stattdessen den “Zwischenspeicher aktualisieren”-Link oben verwenden, um ohne einen Neustart eine Aktualisierung durchzuführen.
 capability_map=Forschungsnetz
 term_capitalized=Begriff
+pause=pause
```

Comment:
Is "pause" the same in German and English?

Reply:
I'm not an expert, but seems likely 
<img width="1167" alt="Screen Shot 2020-06-26 at 9 06 49 AM" src="https://user-images.githubusercontent.com/10748475/85872059-7e455980-b78c-11ea-96e0-e6a05893e5ea.png">

Reply:
yes :)